textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/neextractor/NEExtractorTestConstants.java

```diff
@@ -0,0 +1,151 @@
+package edu.uci.ics.textdb.dataflow.neextractor;
+
+import edu.uci.ics.textdb.api.common.*;
+import edu.uci.ics.textdb.common.constants.SchemaConstants;
+import edu.uci.ics.textdb.common.field.*;
+
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by Sam on 16/4/27.
+ */
+public class NEExtractorTestConstants {
+
+
+    public static final String SENTENCE_ONE ="sentence_one";
```

Comment:
Add a space after "=". Treat it as a general comment.  You should be able to do it easily using your IDE to reformat YOUR OWN files.